### PR TITLE
bot and interface for ubiquity twitter 

### PR DESCRIPTION
**Pull request by ManyRios**
PR for https://github.com/ubiquity/bounty-bot/issues/56 https://github.com/ubiquity/bounty-bot/issues/55

I made a new PR this time

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,52 +1,53 @@
...
-    "smee-client": "^1.2.2",
-    "source-map-support": "^0.5.21",
-    "ts-jest": "^26.4.4",
-    "typescript": "^4.8.4"
-  },
-  "engines": {
-    "node": ">= 10.13.0"
-  }
-}
+{
```
Comment (pavlovcik) [inline package.json:1]: lol

Comment (ManyRios) [reply package.json:1]: ....

Comment (ManyRios) [reply package.json:1]: One More Time :/

**Review by Steveantor — CHANGES_REQUESTED** [review]

File: src/configs/twitterauth.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Credentials for twitter account 
+export const TW_PORT: string = process.env.PORT || "";
+export const TW_API_KEY: string = process.env.API_KEY || "";
+export const TW_API_KET_SECRET: string = process.env.API_KET_SECRET || "";
+export const TW_ACCESS_TOKEN: string = process.env.ACCESS_TOKEN || "";
+export const TW_ACCESS_TOKEN_SECRET: string = process.env.ACCESS_TOKEN_SECRET || "";
```
[the remark this answers is not in the document]
Comment (ManyRios) [reply src/configs/twitterauth.ts]: Done

**Review by pavlovcik — CHANGES_REQUESTED** [review]
Do not replace the entire `package.json` file. Revert and only introduce changes via `yarn`

**Comment by ManyRios** [comment]
Ok... You will payme for this? Or not?

**Comment by pavlovcik** [comment]
> Ok... You will payme for this? Or not?

https://github.com/ubiquity/bounty-bot/issues/56#issuecomment-1366329675

**Comment by ManyRios** [comment]
Ok give me a chance, I'm not in home

**Comment by ManyRios** [comment]
I'll be waiting for your feedback once again

File: package.json (hunk 1 above)
Comment (pavlovcik) [reply package.json:1]: @0xcodercrane do you have any idea what's happening here? @ManyRios reverted the file and then added the dependencies correctly, presumably, using `yarn add` yet Git is considering the entire file to have changed? 

@ManyRios did you change the permissions on the file or something? I have no idea why this is happening.

**Comment by pavlovcik** [comment]
I asked @0xcodercrane to take a quick look because I can't understand what's going on with the `package.json`

Aside from that, can you run a test with the twitter bot you created as part of the other related bounty?

**Comment by ManyRios** [comment]
Yeah sure!

File: package.json (hunk 1 above)
Comment (ManyRios) [reply package.json:1]: No I do not touch permissions and I used yarn for the dependencies.

**Comment by ManyRios** [comment]
> I asked @0xcodercrane to take a quick look because I can't understand what's going on with the `package.json`
> 
> Aside from that, can you (then provide instructions so that I can verify) run a test with the twitter bot you created as part of the other related bounty?

Only thing you need is all the credentials for your account. run the app and try to send a message 

Screenshot_1

**Comment by pavlovcik** [comment]
Shouldn't we be using some type of Twitter "app" or "bot" special service account for the bot? I haven't done the research so I'm not entirely sure, but I have seen "bot" accounts (recognized officially by Twitter's UI) floating around.

**Comment by ManyRios** [comment]
Yep. A developer account for twitter... Nothing else.

I already made a account email for that twitter account the past week only if you dont have one for it

**Comment by pavlovcik** [comment]
We have twitter.com/ubiquitydevpool made for this!

**Comment by ManyRios** [comment]
Great.  I only did it just in case, now just with that account you have to ask twitter for a developer account

**Comment by ManyRios** [comment]
Or send to me your user and password to ask for it

**Comment by pavlovcik** [comment]
> 

I'm working on it now, but the verification email isn't coming in so just give me a little time to figure this out.

**Comment by pavlovcik** [comment]
Might be better to give up that new account you made because Twitter email verification isn't working now. I tried two different domains and checked on the DNS level if any emails came in. None did.

**Comment by ManyRios** [comment]
Ok let me know what can we do then

**Comment by pavlovcik** [comment]
> Might be better to give up that new account you made

**Comment by ManyRios** [comment]
Via telegram? Thats my personal dev account

**Comment by pavlovcik** [comment]
> I already made a account email for that twitter account the past week only if you dont have one for it

Sounded like you wanted to give it up. If you don't want to then we can wait til Twitter fixes its email verification.

**Comment by ManyRios** [comment]
Not at all I can share you my credentials via telegram

**Comment by ManyRios** [comment]
> > I already made a account email for that twitter account the past week only if you dont have one for it
> 
> Sounded like you wanted to give it up. That is what the other bounty was for. If you don't want to then we can wait til Twitter fixes its email verification.

Nothing yet?

**Comment by ManyRios** [comment]
....????

**Comment by ManyRios** [comment]
another test for you 

Screenshot_4
Screenshot_5

**Comment by pavlovcik** [comment]
Unfortunately the amount of time and attention this pull request has cost is not worth it for me to continue. Further, it still is not passing the CI/CD tests, as well as the ongoing issue with your modified package.json.

**Pull request closed without merging**